mcp_zammad/server.py

```
@@ -2041,6 +2041,61 @@ def _build_stats_result(
+        Raises:
+            ValueError: If the response does not contain the expected overview links.
+        """
+        data = client.graphql("{ ticketOverviews(ignoreUserConditions: true) { link ticketCount } }")
+        overviews = data.get("ticketOverviews") or []
+        counts = {
+            item.get("link"): int(item.get("ticketCount") or 0)
+            for item in overviews
+            if isinstance(item, dict) and item.get("link")
+        }
+        if "all_open" not in counts:
+            raise ValueError(f"GraphQL overviews payload misses 'all_open': {counts!r}")
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)